cli/Makefile

```diff
@@ -217,14 +217,12 @@ publish-turbo:
 
 	# Split packing from the publish step so that npm locates the correct .npmrc file.
 	npm pack $(CLI_DIR)/../packages/turbo --pack-destination=$(CLI_DIR)/../
-	npm pack $(CLI_DIR)/../packages/turbo-ignore --pack-destination=$(CLI_DIR)/../
```

Comment:
We publish turbo-ignore independently

Reply:
I'm not ready to make stability guarantees for this but it feels like it is getting much closer.